Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs

```diff
@@ -2032,6 +2032,78 @@ public void Devices_TouchesBecomeStationaryWhenNotMovedInFrame()
         Assert.That(device.allTouchControls[0].phase.ReadValue(), Is.EqualTo(PointerPhase.Stationary));
     }
 
+    [Test]
+    [Category("Devices")]
+    public void Devices_TouchesWithSameIdDontGetStuck()
+    {
+        ////FIXME: Fails - touches stuck in Stationary phase
+        /// While it's not recommended for two different touches to share an id, it shoudn't get stuck in Stationary phase
+        /// Can we add checks for Development build ?
+        var device = InputSystem.AddDevice<Touchscreen>();
+
+        InputSystem.QueueStateEvent(device,
+            new TouchState
+        {
+            phase = PointerPhase.Began,
+            touchId = 0,
+        });
+
+        InputSystem.QueueStateEvent(device,
+            new TouchState
+        {
+            phase = PointerPhase.Ended,
+            touchId = 0,
+        });
+
+        InputSystem.QueueStateEvent(device,
+            new TouchState
+        {
+            phase = PointerPhase.Began,
+            touchId = 0,
+        });
+
+        InputSystem.QueueStateEvent(device,
+            new TouchState
+        {
+            phase = PointerPhase.Ended,
+            touchId = 0,
+        });
+        InputSystem.Update();
+        InputSystem.Update();
+
+        Assert.That(device.activeTouches.Count, Is.EqualTo(0));
+        Assert.That(device.allTouchControls[0].phase.ReadValue(), Is.EqualTo(PointerPhase.None));
+    }
+
+    [Test]
+    [Category("Devices")]
+    public void Devices_TouchesWithWrongTimestampCorrectlyRecognized()
```

Comment:
Same here. Prefix with TODO_.

Reply:
Alternatively we could also adopt a FIXME_ prefix for tests that isolate known bugs (unlike the more general "this still needs to be implement" kind of thing that the TODO_ tests generally revolve around).